src/controllers/product.controller.js

```diff
@@ -0,0 +1,32 @@
+
+import { ProductService } from "../services/product.service";
+import { uploadPhoto } from "../utils/cloudinary.util";
+
+
+
+export const createProduct = async (req, res) => {
+  try {
+    const images = [];
+    for (let i = 0; i < req.files.images.length; i++) {
+      const { url } = await uploadPhoto(req, res, req.files.images[i]);
```

Comment:
Which  file are allowed to upload ? what if i upload pdf ? you need to validate this.